octobot_evaluators/evaluator/realtime_evaluator.py

```diff
@@ -36,3 +37,26 @@ def get_symbol_candles(self, exchange_name: str, exchange_id: str, symbol: str,
         except ImportError:
             self.logger.error(f"Can't get candles manager: requires OctoBot-Trading package installed")
 
+    def _get_tentacle_registration_topic(self, all_symbols_by_crypto_currencies, time_frames, real_time_time_frames):
+        currencies, symbols, _ = super()._get_tentacle_registration_topic(all_symbols_by_crypto_currencies,
+                                                                          time_frames,
+                                                                          real_time_time_frames)
+        to_handle_time_frames = []
+        if self.time_frame is None:
+            self.logger.error("Missing self.time_frame value, impossible to initialize this evaluator.")
+        else:
+            ideal_time_frame = TimeFrames(self.time_frame)
+            to_handle_time_frame = ideal_time_frame
```

Comment:
I think this logic should be moved elsewhere. It's not something only realtime strategy related

Reply:
Do you have an idea where ?

Reply:
Hum I was thinking about trading but it wont be nice to have an import here. Commons is not really concerned about it. I don't know where. Somewhere in Evaluators, in an utility or common file.

Reply:
You added a new util file in this PR, it's a good place to put this I think